docker-compose.yaml

```diff
@@ -1,4 +1,20 @@
 version: '3.6'
 services:
 
-# TODO: Specify the services here. Refer cwl-wes
\ No newline at end of file
+    drs:
+        build:
+            context: .
+            dockerfile: Dockerfile
+        restart: unless-stopped
+        links: 
+            - mongodb
+        command: bash -c "cd /app/drs_filer; python app.py"
+        ports:
+            - "7777:8080"
```

Comment:
Perhaps use some different ports than the ones for cwl-WES, proWES, proTES, TEStribute, so that there won't be clashes if deployed locally with `docker-compose`. Alternatively, keep the original ports. This will mean clashes, too, but at least you are clashing on the default ports.

Reply:
Changing it to 8888:8080? I don't understand the second part. Do you mean doing like this 8080:8080?

> Alternatively, keep the original ports. This will mean clashes, too, but at least you are clashing on the default ports.